### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by tkrille**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,51 +4,58 @@
...
-- Add Flyway migration to replace method-based scopes
-
-    The migration will remove all old, method-based scopes from all well-known
-    clients and then add the new scopes `ADMIN` and `ME`. See the
-    [migration notes] (docs/Migration.md#from-22-to-30) for further details.
-
+- Distribution artifacts have been completely dropped. The `.war` file contains
+  all needed files and assets now.
+- Use sensible defaults for logging. Default level is now error, Spring stuff
+  is logging warnings, and OSIAM log on info level.
```
Comment (wallner) [inline CHANGELOG.md]: "and OSIAM logs", should be third person, right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,51 +4,58 @@
...
   database, an initial admin user will be created. The details of the client and
   user will be logged. This removes the creation of initial data during the
   database setup.
-- Migrate to BCrypt hashes
-  When users login, their passwords will be updated to the new BCrypt hashes.
-  Until the Release 4.0 the old sha password hashes will be supported.
-  After OSIAM 4.0 was released the old password hashes are not usable anymore.
+- Migrate to BCrypt hashes. When users login, their password will be updated to
+  a BCrypt hash. Until the Release 4.0 the old SHA-512 password hashes will be
+  supported. After OSIAM 4.0 was released the old password hashes are not usable
```
Comment (wallner) [inline CHANGELOG.md]: "After OSIAM 4.0 is released"

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,51 +4,58 @@
...
   the database, an example client will be created. If there are no users in the
   database, an initial admin user will be created. The details of the client and
   user will be logged. This removes the creation of initial data during the
   database setup.
-- Migrate to BCrypt hashes
-  When users login, their passwords will be updated to the new BCrypt hashes.
-  Until the Release 4.0 the old sha password hashes will be supported.
-  After OSIAM 4.0 was released the old password hashes are not usable anymore.
+- Migrate to BCrypt hashes. When users login, their password will be updated to
+  a BCrypt hash. Until the Release 4.0 the old SHA-512 password hashes will be
```
Comment (wallner) [inline CHANGELOG.md]: don't capitalize "release"

**Comment by tkrille** [comment]
Thanks for your comments @wallner. I've updated the PR and reworded the whole entry about BCrypt.

**Comment by wallner** [comment]
Excellent, thank you.

**Pull request merged**